heareval/tasks/util/audio.py

```diff
@@ -90,14 +90,17 @@ def audio_stats_wav(in_file: Union[str, Path]):
 
 
 def audio_dir_stats_wav(
-    in_dir: Union[str, Path], out_file: str, exts: List[str] = [".wav", ".mp3", ".ogg"]
+    in_dir: Union[str, Path], out_file: str, exts: Optional[List[str]] = None
 ):
     """Produce summary by recursively searching a directory for wav files"""
+    if exts is None:
+        exts = [".wav", ".mp3", ".ogg"]
 
     # Filter the files in the directory for the required extensions
     audio_paths = list(
         filter(
-            lambda audio_path: audio_path.suffix.lower() in map(str.lower, exts),
+            lambda audio_path: audio_path.suffix.lower()
+            in map(str.lower, exts),  # type: ignore
```

Comment:
Did black format this? This seems like weird formatting for a line break in the middle of an expression

Reply:
I split it across lines because we have to disable mypy for this line, but then the line is too long for flake8. ah my life